src/meson.build

```diff
@@ -149,12 +149,35 @@ fwupdoffline = executable(
 )
 endif
 
-resources_src = gnome.compile_resources(
-  'fwupd-resources',
-  'fwupd.gresource.xml',
-  source_dir: '.',
-  c_name: 'fu'
-)
+# this requires a new enough meson version or for the fix to be backported
+if get_option('gresource_quirks')
```

Comment:
Doesn't this need a guard on meson version?

Reply:
or maybe an `error()`? It would be weird to specify the non-default option and then for it silently do nothing